KsApi/models/graphql/GraphMutationPostCommentEnvelope.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public struct GraphMutationPostCommentEnvelope: Decodable {
```

Comment:
Let's make this type `internal` to be sure it's not exposed to the app.

Reply:
Updated